pkg/reconciler/apis/apiexport/apiexport_reconcile.go

```diff
@@ -130,6 +131,7 @@ func (c *controller) createIdentitySecret(ctx context.Context, clusterName logic
 	if err != nil {
 		return err
 	}
+	secret.Annotations[logicalcluster.AnnotationKey] = clusterName.String()
```

Comment:
Just making sure: this is only needed for `logging.WithObject` to include the cluster name?

Reply:
Yes. I'm open to better ideas :|